src/pde_solve.jl

```diff
@@ -59,5 +66,87 @@ function DiffEqBase.solve(
 
     Flux.train!(loss, ps, data, opt; cb = cb)
 
-    save_everystep ? iters : u0(X0)[1]
+
+    if give_limit == false
+        save_everystep ? iters : u0(X0)[1]
+    else
+        A = prob.kwargs.data.A
+        u_domain = prob.kwargs.data.u_domain
+
+        ## UPPER LIMIT
+        m1 = trajectories_upper
+        sdeProb = SDEProblem(μ , σ , X0 , prob.tspan)
+        ensembleprob = EnsembleProblem(sdeProb)
+        sim = solve(ensembleprob, sdealg, ensemblealg, dt=dt,trajectories=m1,adaptive=false)
+        function sol_high()
+            Uo = []
+            for u in sim.u
+                xsde = u.u
+                U = g(xsde[end])
+                u = u0(X0)[1]
+                for i in length(ts):-1:3
+                    t = ts[i]
+                    _σᵀ∇u = σᵀ∇u[i-1](xsde[i-1])
+                    dW = sqrt(dt)*randn(d)
+                    U = U .+ f(xsde[i-1], U, _σᵀ∇u, p, t)*dt .- _σᵀ∇u'*dW
```

Comment:
are the dW supposed to be pulled from the forward pass here?

Reply:
There is nothing specifically written. The symbols are same though. But problem was that if we take the same the trajectories cannot be taken different..

Reply:
oh yes I see, then it'll go right back, so this is quantifying process uncertainty.

Reply:
Should I change it then?

Reply:
no you're good